.github/workflows/stale_labeler.yaml

```diff
@@ -0,0 +1,19 @@
+name: 'Label inactive PRs as stale'
+
+on:
+  schedule:
+    - cron: '0 11 * * *'  # runs at 11am UTC every day => early morning in US
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8  # https://github.com/marketplace/actions/close-stale-issues
+        with:
+          stale-pr-label: 'stale'
+          stale-pr-message: "This PR is being marked as stale because there hasn't been activity in 14 days. It may be closed by a rosdistro maintainer. If this label isn't appropriate, you can ask a maintainer to remove the label and add the 'persistent' label. You can find our community contributing guidelines here: https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md."
```

Comment:
```suggestion
          stale-pr-message: "This PR hasn't been activity in 14 days. If you are still are interested in getting it merged please provide an update. Otherwise it will likely be closed by a rosdistro maintainer following our [contributing policy](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md). It's been labeled "stale" for visibility to the maintainers. If this label isn't appropriate, you can ask a maintainer to remove the label and add the 'persistent' label.
```

Reply:
That's much better 👍